Engine/DataFeeds/SubscriptionDataReader.cs

```diff
@@ -365,7 +365,9 @@ public bool MoveNext()
                         }
                     }
 
-                    if (instance.EndTime < _periodStart)
+                    // To deal with OpenInterest data that only comes in once per day, we will check it against the periodStart date
```

Comment:
comment + `it will later be filtered correctly based on the start time by the SubscriptionFilterEnumerator which is after the FFEnumerator`